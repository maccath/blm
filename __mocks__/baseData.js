const baseData = {
  site: {
    siteMetadata: {
      author: "Site Author",
      description: "Site description",
      title: "Site Title",
    },
  },
  placeholderImage: {
    childImageSharp: {
      fluid: {
        src: "icon.png",
        srcSet: "icon.png 50w, icon.png 100w",
        sizes: "(max-width: 200px) 100vw, 200px",
        base64: "data:image/png;base64,iVBORw0KG",
        aspectRatio: 1,
      },
    },
  },
}

export default baseData
