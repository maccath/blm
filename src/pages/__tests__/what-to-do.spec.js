import React from "react"
import renderer from "react-test-renderer"
import { useStaticQuery } from "gatsby"
import "jest-styled-components"
import baseData from "../../../__mocks__/baseData"

import WhatToDo from "../what-to-do"
import Layout from "../../components/layout"

beforeEach(() => {
  useStaticQuery.mockImplementation(() => baseData)
})

describe("What To Do page", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Layout>
          <WhatToDo />
        </Layout>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
