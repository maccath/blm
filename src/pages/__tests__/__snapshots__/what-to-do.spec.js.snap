// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`What To Do page renders correctly 1`] = `
<div
  style={
    Object {
      "margin": "0 auto",
      "maxWidth": 960,
      "padding": "0 1.0875rem 1.45rem",
    }
  }
>
  <main>
    <h1>
      But what can I do about it?
    </h1>
    <p>
      If you've read this far; you've already done some valuable work. Acknowledging your white privilege and the existence of white supremacist culture is the first step on the journey to being actively anti-racist as opposed to just not being racist.
    </p>
    <p>
      Standing in solidarity with those who are experiencing oppression does not need to be at the expense of other causes. There are many things we can do to stand on the right side of history as it unfolds.
    </p>
    <p>
      I encourage you to keep this in mind as you navigate the world from now on. You may leave here, or stay for some more ideas and resources to put anti-racism into action.
    </p>
    <div
      className="mt-6 flex space-x-4"
    >
      <a
        class="flex-initial rounded-lg bg-white text-purple p-3 no-underline"
        href="/donate"
      >
        I have some spare cash.
      </a>
      <a
        class="flex-initial rounded-lg bg-white text-purple p-3 no-underline"
        href="/learn"
      >
        I care and want to learn more.
      </a>
      <a
        class="flex-initial rounded-lg bg-white text-purple p-3 no-underline"
        href="/amplify"
      >
        I'd like to show solidarity.
      </a>
    </div>
  </main>
  <footer
    className="css-1bxtzlt"
  >
    <p>
      Compiled by 
      <a
        href="http://katy-ereira.co.uk"
      >
        Katy Ereira
      </a>
       /
       
      <a
        href="http://twitter.com/maccath"
      >
        @maccath
      </a>
       | Fist icon by
       
      <a
        href="https://thenounproject.com/cesarreynoso562/"
      >
        Cesar Reynoso
      </a>
    </p>
  </footer>
</div>
`;
